### PR TITLE
adding my nft-price-checker app

### PR DESCRIPTION
**Pull request by mathisfchx**
Here is the address of the bellecour deployed app : 0xe568E468495da3A2502EC6f6b5004144e2A65a51
Here are some viviani debug app (on v7 debug workepool) successfull tasks :
	args = "web2" : 0xc5acfa4ad5aa90f0f60c5c4c5b7bf3253ac34aad87aca733bbc075217b28576d
	args = "web3" : 0xb0b2edeed53f72283c29d2b0704636653fdbf3e73244efeb98d87ff36fed4bcc
And here is the orderHash of some "free" published apporder : 0x5ab21478ead12283f44035e3441beaa7ed3ed40c11eca9d831b917f7f7411d1c

### CODE REVIEW

File: mathisfchx/nft-price-checker/README.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+cat /tmp/iexec_out/result.txt
+cat /tmp/iexec_out/computed.json
+```
+
+### Deploy
+To deploy your app, follow the instructions on the IExec Documentation : https://docs.iex.ec/for-developers/your-first-app
+
+Then, you can run your dApp with the `iexec app run` command (you can add as much parameters and options as you want, follow the SDK and CLI documentation to do so) :  
+```
+iexec app run --watch
```
Comment (james-toussaint) [inline mathisfchx/nft-price-checker/README.md]: Maybe you can add links and/or info of 2 successful tasks over here :)

**Comment by mathisfchx** [comment]
✅  Successful task on bellecour debug (with a new debug image app 0x7C6620fB2e7bC883d5B834f8Cd6B9e62a069f0b0) : https://v7.core.debug-main-pool.bellecour.iex.ec/tasks/0xafed63aad70cdddc4fcd6dcb8e491a450d506a3dc5d433826e0dd763944d0d7d

**Comment by mathisfchx** [comment]
✅  Successful task on bellecour prod (with a new sconified 5.7.2 prod image app 0xD037626C0F37a05eD8713242a61144f521fc4add) : https://v7.core.prod-main-pool.bellecour.iex.ec/tasks/0xbdffc5b7157daf129a85ad8bb363f2762924085fd837669749d5f5d6e0317df8

File: mathisfchx/nft-price-checker/iexec.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "social": {
+    "website": "?",
+    "github": "https://github.com/iExecBlockchainComputing/nft-price-checker"
+  },
+  "logo": "logo.png",
+  "buyConf": {
+    "params": {
+      "iexec_args": "web2"
+    },
+    "tag": "0x0000000000000000000000000000000000000000000000000000000000000000",
```
Comment (PierreJeanjacquot) [inline mathisfchx/nft-price-checker/iexec.json]:
```suggestion
    "tag": "0x0000000000000000000000000000000000000000000000000000000000000001",
```
use `tee` tag

File: mathisfchx/nft-price-checker/deployed.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "app": {
+    "134": "0xe568E468495da3A2502EC6f6b5004144e2A65a51"
```
Comment (PierreJeanjacquot) [inline mathisfchx/nft-price-checker/deployed.json]:
```suggestion
    "134": "0xD037626C0F37a05eD8713242a61144f521fc4add"
```
update app address?

**Review by PierreJeanjacquot — CHANGES_REQUESTED** [review]
see comments + missing apporder for `0xD037626C0F37a05eD8713242a61144f521fc4add`

**Comment by mathisfchx** [comment]
Thanks for the comments @PierreJeanjacquot, +now the apporders are up !

**Review by PierreJeanjacquot — APPROVED** [review]

**Pull request merged**